docs/framework-integrations/svelte.mdx

```diff
@@ -50,7 +50,7 @@ instance can be passed into components as an `uppy` prop. Due to the way Svelte
 handles reactivity, you can initialize Uppy the same way you would with vanilla
 JavaScript.
 
-```html
+```svelte
```

Comment:
Did you validate if Docosorus is still doing syntax highlighting with this change?

Reply:
Quite confident it should

Reply:
I just tried, the syntax highlighting is gone